subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -791,7 +863,14 @@ static psa_status_t convert_from_psa_attributes(const psa_key_attributes_t *key_
 		break;
 #endif
 	default:
-		return PSA_ERROR_NOT_SUPPORTED;
+		/* Ignore the algorithm for the protected ram invalidation kmu slot because
+		 * it will never be used for crypto operations.
+		 */
+		int kmu_slot = CRACEN_PSA_GET_KMU_SLOT(
```

Comment:
If you put `{ }` in the default case, I think this Clang warning goes away

Reply:
I think that you are right, I decided to move the declaration at the top just to follow the spec a bit more closely.